R/snapshot-file.R

```diff
@@ -3,7 +3,7 @@
 #' @description
 #' Whole file snapshot testing is designed for testing objects that don't have
 #' a convenient textual representation, with initial support for images
-#' (`.png`, `.jpg`, `.svg`) and data frames (`.csv`).
```

Comment:
Data frames are specifically supported, so can you please add an additional clause for text files?

Reply:
:+1: